plugins/modules/redfish_command.py

```diff
@@ -239,8 +239,19 @@
     type: bool
     default: false
     version_added: 3.7.0
+  bios_attributes:
+    required: false
+    description:
+      - BIOS attributes that needs to be verified in the given server
+    type: dict
+    version_added: 6.1.0
```

Comment:
```suggestion
    version_added: 6.3.0
```
(Since the 6.3.0 release is on Tuesday, 6.4.0 is probably more likely.)

Reply:
Changed to 6.4.0. Thanks!